code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -104,15 +104,27 @@
 
 		if("howl", "howls")
 			var/M = handle_emote_param(param) //Check to see if the param is valid (mob with the param name is in view).
-			message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
-			playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
-			m_type = 2
+			if(miming)
+				message = "<B>[src]</B> tilts \his head back as if to howl."
+				m_type = 1
+			else
+				if(!muzzled)
+					message = "<B>[src]</B> howls[M ? " at [M]" : ""]!"
+					playsound(loc, 'sound/goonstation/voice/howl.ogg', 100, 0, 10)
+					m_type = 2
+				else
+					message = "<B>[src]</B> makes a strange muffled noise."
```

Comment:
"makes a very sharp and loud noise".

Since howling and screaming are similar volume, and the flavortext of screaming is "makes a very loud noise".